src/webargs/flaskparser.py

```diff
@@ -117,6 +116,18 @@ def get_default_request(self):
         return flask.request
 
 
+class FlaskAsyncParser(FlaskParser, asyncparser.AsyncParser):
```

Comment:
I'd like it if we could, in the future, support async flask views in `FlaskParser` itself, but I foresee issues with it (primarily that `FlaskParser.parse` needs to be synchronous). This approach of adding a new parser class is easier to do now and gets us some support. We can always revisit in v9 or later.
👍 for thinking of this approach!

I need to think about this more, but I'm pretty sure this empty class is correct.
The MRO should be `[FlaskParser, AsyncParser, Parser, object]` from this, and that seems like it will behave correctly.

Reply:
@slallema said in #662 they did it by modifying FlaskParser. Details about how would be welcome.